File: Sc.Models/Dtos/Sso/SsoUserInfoDto.cs
using Sc.Models.Dtos.Nomenclatures.Settlements;

namespace Sc.Models.Dtos.Sso
{
    public class SsoUserInfoDto
    {
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string FullName { get; set; }
        public int? DistrictId { get; set; }
        public DistrictDto District { get; set; }
        public int? MunicipalityId { get; set; }
        public MunicipalityDto Municipality { get; set; }
        public int? SettlementId { get; set; }
        public SettlementDto Settlement { get; set; }
        public string Address { get; set; }
    }
}

File: WebApplication/src/auth/dtos/sign-up/sso-user-info.dto.ts
import { DistrictDto } from "src/nomenclatures/dtos/settlements/district.dto";
import { MunicipalityDto } from "src/nomenclatures/dtos/settlements/municipality.dto";
import { SettlementDto } from "src/nomenclatures/dtos/settlements/settlement.dto";

export class SsoUserInfoDto {
    firstName: string;
    middleName: string;
    lastName: string;
    fullName: string;
    districtId: number;
    district: DistrictDto;
    municipalityId: number;
    municipality: MunicipalityDto;
    settlementId: number;
    settlement: SettlementDto;
    address: string;
}